README.md

```diff
@@ -385,8 +385,6 @@ decoder.decode(model2_message_value)
 
 The following validations are supported:
 
-- The size of the value for a fixed type field.
-- The value for an enum type field is in the declared set of values.
```

Comment:
Upgrade info to follow should make it clear that these validations are now enforced as part of coercion -- not something that has been removed.